react-multiple-components/my-app/src/H2.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  label: string;
+};
+
+export function H2({ label }: Props) {
```

Comment:
I'm not thrilled by this name. It's too HTML-ish. What if you decide to make it an `h1` or `div` instead? A name like "header" or similar would be better.